web/controllers/screeningController/index.ts

```diff
@@ -301,3 +303,106 @@ export async function updateScreenerByMailHandler(
         next();
     }
 }
+
+/**
+ * @api {POST} /courses getCourses
+ * @apiVersion 1.0.1
+ * @apiDescription
+ * 
+ * Retrieves the first 20 courses that match the specified filters.
+ * 
+ * 
+ * Only screeners with a valid token in the request header can use the API.
+ *
+ * @apiName getCourses
+ * @apiGroup Screener
+ *
+ * @apiUse Authentication
+ *
+ * @apiExample {curl} Curl
+ * curl -k -i -X POST -H "Token: <AUTHTOKEN>" https://dashboard.corona-school.de/api/screening/courses
```

Comment:
The current url is `api.corona-school.de`, not `dashboard`. (The old docs in this file have the same issue.)  
Also method should be GET

Reply:
Then maybe the host should be omitted?